servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -39,3 +39,37 @@ vm.max_map_count = 262144
 ----
 $ sysctl vm.max_map_count=262144
 ----
+
+
+[[updating-container-registry]]
+== Updating the container registry
+
+[NOTE]
+====
+If you are running {product-title} 3.11 on-premise, follow these steps to configure access to https://registry.redhat.io[registry.redhat.io].
```

Comment:
should this be premises? 

https://brians.wsu.edu/2016/05/30/premise-premises/

Reply:
I believe on-premise is a term which has come to be accepted despite it being incorrect.  This PR has also been merged so we should leave this up to @geekspertise and others to determine whether it needs updating in a new PR.

Reply:
@knrc is right - the usage is technically incorrect but accepted by CCS because of its widespread industry acceptance.

Reply:
We have a style guide, and it currently says "on-premise".